# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: third_party/rust/ash/tests/constant_size_arrays.rs
# Commit: ef5dc3e04e5f
# Full Hash: ef5dc3e04e5f271eea0636ab3a495e95cc912f1d
# Author: Dzmitry Malyshau <dmalyshau@mozilla.com>
# Date: 2021-09-04 09:40:24
# Regressor Bug: 1726626
# File Overlap Count: 3
# Description:
#   Bug 1726626 - Move gfx/wgpu into a 3rd party dependency r=jgilbert,bholley
#   
#   This update makes wgpu a vendored dependency instead of having it in gfx/wgpu.
#   
#   ## Notes
# ==============================================================================

diff --git a/third_party/rust/ash/tests/constant_size_arrays.rs b/third_party/rust/ash/tests/constant_size_arrays.rs
--- a/third_party/rust/ash/tests/constant_size_arrays.rs
+++ b/third_party/rust/ash/tests/constant_size_arrays.rs
@@ -19,16 +19,15 @@ fn assert_struct_field_is_array() {
         blend_constants,
         ..Default::default()
     };
 }
 
 #[test]
 #[allow(dead_code)]
 fn assert_ffi_array_param_is_pointer() {
-    use ash::version::DeviceV1_0;
     // don't run it, just make sure it compiles
     unsafe fn dummy(device: &ash::Device, cmd_buffer: ash::vk::CommandBuffer) {
         let blend_constants: [f32; 4] = [0.0, 0.0, 0.0, 0.0];
 
         device.cmd_set_blend_constants(cmd_buffer, &blend_constants);
     }
 }